content/docs/machine-learning/creating-a-new-kubeflow-notebook.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 4
+title: Creating a new Kubeflow notebook
+description: Creating a new notebook in Civo's machine learning Kubeflow service
+---
+
+<head>
+  <title>Creating a new Kubeflow notebook | Civo Documentation</title>
+</head>
+
+To create a new Kubeflow notebook, you will need to provide a name, select a development environment, and make a few architectural decisions. All of these are explained below.
+
+## Notebook options
```

Comment:
Overall I think its super helpful for docs readers to understand the overall flow, something like:

- Kubeflow first creates a StatefulSet in which the pods for our Notebook Server is created
- A pod is created for our Notebook Server kubeflow-series-part2-0 which is where our code runs
- A dedicated Persistent Volume Claim (we talk more about this soon) is attached to this pod which is where our home directory when using the notebook server resides
- Our Notebook Server uses the image we put in and starts a container in the pod

Reply:
I think this is all very useful stuff, you're absolutely right. I don't see it in your MR yet though?

Additionally, the "Accessing your Notebook Server" section seems very thin. What will the user see when they click the Connect button? What will they need to do next? Where can we point them, if it's not something we want to cover in the docs?

Reply:
> I think this is all very useful stuff, you're absolutely right. I don't see it in your MR yet though?

Ah, okay added this, I wasn't sure if we wanted this